templates/admin.html

```diff
@@ -18,5 +18,22 @@ <h2>Home page content</h2>
 		<br />
 	</form>
 </section>
+<section>
+	<h2>Print All ID Cards</h2>
+	<form action=/cards/all>
+		<h3>Select semester for which to generate ID cards:</h3>
+		<select name="semester" size="{{ semesters|length }}">
+			{% for semester in semesters %}
+			<option value={{ semester }}{% if semester == current_semester%} selected="selected"{%endif%}>{{ semester }}</option>
```

Comment:
Can you make the spacing here consistent (i.e. `{% if ... current_semester%}` → `{% if ... current_semester %}`; `{%endif%}` → `{% endif %}`)?